.vib/prestashop/cypress/cypress/integration/prestashop_spec.js

```diff
@@ -0,0 +1,102 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+// Note: Interaction with the UI is preferred over cy.visit() as
+// PrestaShop requires to provide a token within the URL or else
+// a security warning page is shown.
+
+it('allows login in and out', () => {
+  cy.login();
+  cy.contains('Activity overview');
+
```

Comment:
Is there any reason for adding this empty line?

Reply:
Not really haha, just removed the empty lines now from the file :)